app/controllers/users_controller.rb

```diff
@@ -1,9 +1,11 @@
 class UsersController < ApplicationController
   def index
     @users = User.all
+    @posts = @user.posts
```

Comment:
- [ ] i get this error undefined method for posts  kindly fix, It worked when I replaced with ` @user = User.first` you can try other things
![Screenshot from 2023-04-21 17-00-56](https://user-images.githubusercontent.com/66097371/233682657-fcbbbbd6-4e3d-4bb3-ba57-cb37113db4df.png)